src/index.ts

```diff
@@ -1,12 +1,13 @@
 import connect from './connection';
-import { AMQPOptions } from './adapters/amqp-node';
+import { AMQPOptions, AMQPQueueOptions } from './adapters/amqp-node';
 import { Logger } from './logger';
 import { MessageHandler } from './message';
 
 export interface CreateServiceOptions {
   serviceName: string;
   logger: Logger;
   connectOptions: AMQPOptions;
+  queueOptions?: AMQPQueueOptions;
```

Comment:
Could you please update the readme file with new option?

Reply:
Done